app.js

```diff
@@ -0,0 +1,11 @@
+let calculators = ko.observableArray([new Calculator()]);
+
+function addCalculator() {
```

Comment:
This code is not specific to the calculator itself. This is for adding new calculators. A calculator should not know that it can be added/removed from the page. That is a concern of the wider application, not the calculator.